### PR TITLE
Milestone two

### PR DESCRIPTION
**Pull request by Ispirett**
Milestone ready to review.
Addressed issues from previous review.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,18 @@
 import React from 'react';
```
Comment (shloch) [inline src/App.js]: please relocate this file `App.js`  into   `src/components/` folder

File: src/components/BookList.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React,{useEffect} from 'react'
```
[the remark this answers is not in the document]
Comment (Ispirett) [reply src/components/BookList.js]: > this is a container element that will contain books, please relocate this file (`BookList.js`) into `src/components/` folder

Hey man thanks for review :) The BookList.js file is already in the src/components folder.Is there another file you want me to put there?

[the remark this answers is not in the document]
Comment (Ispirett) [reply src/components/BookList.js]: I am @Ispirett

[the remark this answers is not in the document]
Comment (shloch) [reply src/components/BookList.js]: sorry i meant `src/containers/` folder

**Pull request merged**